src/lib.rs

```diff
@@ -162,9 +162,16 @@ impl SteamDir {
     ///
     /// [See the struct docs][Self#example] for an example
     pub fn locate() -> Result<Self> {
-        let path = locate::locate_steam_dir()?;
+        let paths = locate::locate_steam_dir()?;
 
-        Self::from_dir(&path)
+        Self::from_dir(&paths[0])
```

Comment:
This got changed from returning a `ValidationError` for a missing directory to panicking when missing. You can get the old behavior back with something like

```rust
let path = paths.get(0).ok_or_else(/* missing dir validation error */)?;
Self::from_dir(&path)
```